src/main.c

```diff
@@ -112,7 +135,7 @@ int exec_cmdline_arg(int argcin, char *argvin[])
 
 	sprintf(header, "%s %s\n%s\n\n", PROCESS, VERSION_NUM, COPYRIGHT);
 
-	while ((i = getopt(argcin, argvin, "?:a:b:c:d:ef:gH:hi:jL:lm:NnP:p:qRr:s:t:U:u:VvXx")) != INVALID_VALUE) {
+	while ((i = getopt(argcin, argvin, "?:a:b:c:d:ef:gH:hi:jL:lm:NnP:p:qRr:s:t:U:u:VvXxYyzZoOTM")) != INVALID_VALUE) {
```

Comment:
It is much easier to follow if these are placed in alphabetical order (capital first, then lower case).

Reply:
> It is much easier to follow if these are placed in alphabetical order (capital first, then lower case).

Ok! Please consider all the new swicth's names as **completely** temptative.